configure.ac

```diff
@@ -367,12 +369,15 @@ AS_IF([test "x$with_netcdf" != "xno"],
         AC_MSG_CHECKING([if we can compile NetCDF with C++])
         # Note netcdf_c++ needed
         NCLIB=`$NCCONF --libs | sed s/-lnetcdf/-lnetcdf_c++\ -lnetcdf/`
+        LDFLAGS="$LDFLAGS -Wl,-rpath,`nc-config --prefix`/lib"
```

Comment:
Unfortunately, I think this needs to be a bit more complicated, as it might be `lib64` on some systems. I think NetCDF 4.4.1.1 now has `nc-config --libdir` which tells you the correct path. Otherwise, we need to parse `nc-config --libs` and extract the path on `-L`.